packages/chart/src/components/BoxPlot/BoxPlot.tsx

```diff
@@ -5,6 +5,7 @@ import ConfigContext from '../../ConfigContext'
 import ErrorBoundary from '@cdc/core/components/ErrorBoundary'
 import { colorPalettesChart } from '@cdc/core/data/colorPalettes'
 import { handleTooltip, calculateBoxPlotStats, createPlots } from './helpers/index'
+import { v4 as uuidv4 } from 'uuid'
```

Comment:
Is this needed in the file or was it previously removed?

Reply:
Removed. We don't need it.